src/page/ListPage/TodoPage.jsx

```diff
@@ -24,8 +29,14 @@ export const TodoPage = () => {
   };
 
   const updateTodo = (id, newText, completed) => {
-    const updatedTodos = todos.map(todo =>
-      todo.id === id ? { ...todo, text: newText ?? todo.text, completed: completed ?? todo.completed } : todo
+    const updatedTodos = todos.map((todo) =>
```

Comment:
at some point in the video you make a comment related to filtering the `todos` and that this is ok just cause this is a tiny project

⚠️ NOP

this **is** the way

<img src="https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExNDc2eW9mZXIxMmZ0bmI0ejV6cXd0ajRjMXN6Zmg0d2d4bGppaDlxdCZlcD12MV9naWZzX3NlYXJjaCZjdD1n/6UFgdU9hirj1pAOJyN/giphy.gif" width=200/>

do not create additional states (you mentioned a `useReducer()`) using derived state from your actual state is perfect!

good job